reactor-core/src/main/java/reactor/core/publisher/FluxTimeout.java

```diff
@@ -266,7 +286,7 @@ void handleTimeout() {
 			if (other == null) {
 				super.cancel();
 				actual.onError(new TimeoutException("Did not observe any item or terminal signal within "
-						+ timeoutDescription + " (and no fallback has been configured)"));
+						+ timeoutDescription + " in '" + name + "' (and no fallback has been configured)"));
```

Comment:
...and accommodate for `null` here... You could actually prepare the message in the constructor, once.